airbyte-integrations/bases/standard-test/standard_test/utils/connector_runner.py

```diff
@@ -99,8 +105,10 @@ def run(self, cmd, config=None, state=None, catalog=None, **kwargs) -> Iterable[
         logs = self._client.containers.run(
             image=self._image, command=cmd, working_dir="/data", volumes=volumes, network="host", stdout=True, stderr=True, **kwargs
         )
-        logging.info("Running docker, folders: %s", volumes)
+        logging.info("Docker run: \n%s\ninput: %s\noutput: %s", cmd, self.input_folder, self.output_folder)
+
+        with open(str(self.output_folder / "raw"), "wb+") as f:
```

Comment:
How can we prevent this from filling up disk space over time? maybe output should live in `/tmp` by default?

Reply:
output and input folders are in `/tmp` by default, we take base folder from 
`tmp_path` fixture (builtin pytest fixture)
```python
    """Return a temporary directory path object which is unique to each test
    function invocation, created as a sub directory of the base temporary
    directory.
    """

```

Reply:
so the output looks like this
```
INFO     root:connector_runner.py:108 Docker run:
read --config tap_config.json --catalog catalog.json
input: /tmp/pytest-of-root/pytest-141/test_two_sequential_reads_inpu0/run_1/input
output: /tmp/pytest-of-root/pytest-141/test_two_sequential_reads_inpu0/run_1/output
```

Reply:
great!